### ISSUE TITLE
install getdns node

### ISSUE THREAD

**Issue by fturiot** [getdnsapi/getdns-node#42]
Hi,
Where load c++ librarie for install ?
regards

**Comment by joelpurra** [comment]
@fturiot: that depends on your operating system, compiler, local settings, etcetera. The easiest way is to use a package manager, since it will use the default library/header file locations on disk. 

Examples include [`apt`](https://en.wikipedia.org/wiki/APT_(software)) to install [`libgetdns-dev`](https://packages.debian.org/stable/libgetdns-dev), [`brew`](https://brew.sh/) to install [`getdns`](https://formulae.brew.sh/formula/getdns), and so on [for a variety of systems](https://pkgs.org/search/?q=getdns). (Unfortunately I am not using Windows, can't help there.)

```shell
# NOTE: example assumes a Debian-based system. You may need additional tools or library dependencies.
sudo apt install libgetdns-dev                        
```

When all files are in their default locations, installation/compilation should work.

```shell
npm install getdns
```

You can also configure your compiler to use custom paths. There are examples of this in #38 and #39, for custom builds from `getdns` source.

```shell
LIBRARY_PATH="/your/getdns/build" CPLUS_INCLUDE_PATH="/your/getdns/build" npm install getdns
```

Did that help?

---

It would of course be even better for `getdns-node` to fix #29, to simplify installation, but that is unlikely to happen until Someone™ finds time for it.

**Comment by fturiot** [comment]
Thanks solve
I hate windows i'm in linux but i have not understand that's a linux package 
thanks
